bread/layout/components/shell_header.py

```diff
@@ -80,3 +62,39 @@ def __init__(self, platform, company, actions=(), *args, **kwargs):
             role="banner",
             data_header=True,
         )
+
+
+def logo():
+    from django.contrib.staticfiles.storage import staticfiles_storage
+
+    return hg.IMG(
+        src=staticfiles_storage.url("logo.png"),
+        _class="bx--header__name--prefix",
+        style="width: 1.7rem; height; 1.7rem; margin-right: 0.5rem",
+    )
+
+
+def variable_size_header_part(platform, company, searchbar, searchbar_position):
+    return hg.BaseElement(
+        hg.A(
+            logo(),
+            platform,
+            _class="bx--header__name",
+            style="font-weight: 400",  # override carbon design
+            href=hg.F(lambda c: c["request"].META["SCRIPT_NAME"] or "/"),
```

Comment:
In case you wondered about this line: The WSGI protocol allows us to "mount" an application on something else than the root of a domain. Se we can e.g. have https://website.com/app1 and https://website.com/app2 and both running with on the same code and seeing themselfes as beeing on the root. All django URL resolving will work correctly as well with this. But when we use our own constructed URLs (like here, because we just want the root-url of this django instance and we don't know the name of that URL) then we have to take care of it.